rita/src/rita_common/oracle/mod.rs

```diff
@@ -338,8 +339,10 @@ fn update_oracle() {
                                     // an explicit fashion
                                     match serde_json::from_slice::<OracleUpdate>(&new_prices) {
                                         Ok(new_settings) => {
-                                            let dao_settings = SETTING.get_dao();
+                                            let mut dao_settings = SETTING.get_dao_mut();
                                             let use_oracle_price = dao_settings.use_oracle_price;
+                                            dao_settings.wyre_enabled = new_settings.wyre_enabled;
```

Comment:
this should not go in dao settings maybe payment settings, but more probably a place called 'localization settings' that we don't serialize at all. 

Let me think on it some.